source/common/rsa.c

```diff
@@ -6,22 +6,21 @@
 #include <nds/rsa.h>
 #include <nds/system.h>
 
-int swiRSAInitHeapTWL(swiRSAHeapContext_t *ctx, void *heapStart, size_t heapSize);
```

Comment:
I would keep the types as `size_t`. It's guaranteed to be 32-bit wide in the DS, and it's better semantically. A `u32` can be any unsigned value, a `size_t` represents a size.